src/Misc.pm

```diff
@@ -1171,7 +1172,11 @@ sub charSelectScreen {
 
 		my $messageDeleteDate;
 		if ($chars[$num]{deleteDate}) {
-			$messageDeleteDate = TF("\n     -> It will be deleted lefting %s!", $chars[$num]{deleteDate});
```

Comment:
That 'lefting' message, I just copy it from src itself. U can find it.
However, the sentence is weird. It's must be something like "Char can be
deleted on %s"
On Jun 16, 2016 2:53 PM, "Rachel Fish" <notifications@github.com> wrote:

> In src/Misc.pm
> <https://github.com/OpenKore/openkore/pull/97#discussion_r67301246>:
>
> > @@ -1171,7 +1172,11 @@ sub charSelectScreen {
> >
> >  		my $messageDeleteDate;
> >  		if ($chars[$num]{deleteDate}) {
> > -			$messageDeleteDate = TF("\n     -> It will be deleted lefting %s!", $chars[$num]{deleteDate});
>
> What is lefting?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenKore/openkore/pull/97/files/66013f9bc9417ed3d647da1a6f8f9c9e9abdbfe8#r67301246>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AD1K4KFr74ejJrb2fO_TtnAlhdoidJEtks5qMQELgaJpZM4Ib1aI>
> .
>